Tasks/AppCenterDistribute/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -26,8 +26,8 @@
   "loc.input.help.releaseNotesInput": "Release notes for this version.",
   "loc.input.label.releaseNotesFile": "Release notes file",
   "loc.input.help.releaseNotesFile": "Select a UTF-8 encoded text file which contains the Release Notes for this version.",
-  "loc.input.label.distributionGroupId": "Distribution group ID",
-  "loc.input.help.distributionGroupId": "ID of the distribution group the app will deploy to. Leave it empty to use the default group.",
+  "loc.input.label.distributionGroupId": "Destination ID",
+  "loc.input.help.distributionGroupId": "ID of the distribution group or store the app will deploy to. Leave it empty to use the default group.",
```

Comment:
is it easy to change this `loc.input.help.distributionGroupId` and `loc.input.label.distributionGroupId` to `loc.input.help.destinationId` and `loc.input.label.distributionGroupId`?

Reply:
Oh, yeah, just saw the discussion below.

Reply:
@nevalenny I agree with @izikl  that we should update the label names

Reply:
Sure, it's updated automatically on a building, changed as part of the property rename.